tests/apps/crypto/utils.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"os/signal"
+	"strconv"
+	"syscall"
+	"time"
+
+	"github.com/google/uuid"
+)
+
+/**
+This file contains code from tests/e2e/utils (with some simplifications).
```

Comment:
See this comment here. Copying this code was required because this app has a Dockerfile (because it needs the SDK).

Importing `github.com/dapr/dapr` in the Dockerfile was causing issues due to protos being registered again.